docs/cdn/asset-compression.md

```diff
@@ -0,0 +1,35 @@
+# Asset Compression Strategy
+
+### What and why compress assets?
+
+For faster load speeds of your page, static assets are compressed at server, and then uncompressed by an end users browser. This is a contributing factor to SEO scores, but generally good practice to help reduce overall bandwidth usage. 
+
+Those static assets include .js or CSS files, images, and also the generated HTML files of your site.
+
+### How do we compress assets?
+
+Assets are compressed using GZIP and is done via NGINX. When they’re sent as a response to a request they’ll also then be cached at the CDN for faster delivery of that assets to subsequent requests.
+
+The [MIME types](https://en.wikipedia.org/wiki/Media_type) we compress are listed below.
+
+`text/plain, text/css, text/html, application/json, application/javascript, application/x-javascript, text/xml, application/xml, application/xml+rss, text/javascript, application/x-font-ttf, font/opentype, application/vnd.ms-fontobject, image/svg+xml`
+
+Images are compressed and cached separately, as we leverage [Dynamic Image Resizing](https://docs.altis-dxp.com/media/dynamic-images/), the Images are created on the fly, and then cached at the CDN layer for serving the image. Images will attempt to be served via `[WebP](https://developers.google.com/speed/webp/)` format if the browser supports it, falling back to the image default format.
```

Comment:
"if the browser supports [WebP](link...), images will be served in that format, falling..."